src/GovUk.Education.ExploreEducationStatistics.Content.Model/EmbedBlock.cs

```diff
@@ -0,0 +1,24 @@
+﻿#nullable enable
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model;
+
+public class EmbedBlock
```

Comment:
Could we add `created` + `updated` fields here for auditing/best practices?

Reply:
I've added `Created` and `Updated` fields to `EmbedBlock`, but I've not put them in models since they wouldn't currently be used. I think we mainly add these for debugging and in case we need them in the future.